chainer/functions/math/basic_math.py

```diff
@@ -191,12 +191,11 @@ def forward(self, xs):
         self.len = len(xs)
         if len(xs) == 1:
             return xs
-        y = None
-        if intel64.should_use_ideep('>=auto'):
-            bxs = numpy.broadcast_arrays(*xs)
-            if intel64.inputs_all_ready(bxs):
-                y = intel64.ideep.multi_add(bxs)
-        if y is None:
+        if (intel64.should_use_ideep('>=auto')
+                and intel64.inputs_all_ready(*xs)
```

Comment:
`inputs_all_ready` expects list of arrays as list. Just pass `xs`.

Reply:
Oops, my manual merge at reverting c5f9195 is wrong.  Instead e1059f6 should be reverted too.

Reply:
Fixed!